Package.swift

```diff
@@ -29,7 +29,7 @@ let package = Package(
         ),
     ],
     dependencies: [
-        .package(name: "StreamChatTestHelpers", url: "https://github.com/GetStream/stream-chat-swift-test-helpers.git", .exact("0.2.2")),
+        .package(name: "StreamChatTestHelpers", url: "https://github.com/GetStream/stream-chat-swift-test-helpers.git", .exact("0.2.3")),
```

Comment:
Just a note: If there are any breaking changes here, let's also check SwiftUI.

Reply:
Sure, there are no breaking changes at this moment, but let's, for now, bump the minor version when there are any breaking changes and the patch version when there are no breaking changes.